fe/src/redux/userSlice.ts

```diff
@@ -102,6 +135,38 @@ const userSlice = createSlice({
       state.message = action.payload;
       state.messageType = "error";
     });
+    builder.addCase(forgotPassword.fulfilled, (state, { payload }: any) => {
+      // set success message
+      state.isSuccess = true;
+      state.message = "Email successfully sent please check your email";
+      state.messageType = "success";
+      state.isFetching = false;
+    });
+
+    builder.addCase(forgotPassword.rejected, (state, action: any) => {
+      // set error message
+      state.message = action.payload;
+      state.messageType = "error";
+      state.isFetching = false;
+    });
+
+    builder.addCase(forgotPassword.pending, (state, action: any) => {
+      // set error message
+      state.isFetching = true;
+    });
+    builder.addCase(resetPassword.fulfilled, (state, action: any) => {
+      // set error message
+      console.log(action);
```

Comment:
Don't practice putting console.log on repository, In case this will be pushed to production we don't want to expose any data because of this code. If you want to see logs better use Logger tools and can only be applied to dev server or local.